lib/taskana-core-test/src/test/java/acceptance/task/create/CreateTaskAccTest.java

```diff
@@ -0,0 +1,967 @@
+    assertThat(createdTask.getPlanned()).isEqualTo(expectedPlanned);
+    assertThat(createdTask.getState()).isEqualTo(TaskState.READY);
+    assertThat(createdTask.getParentBusinessProcessId()).isNull();
+    assertThat(createdTask.getPriority()).isEqualTo(defaultClassificationSummary.getPriority());
+    assertThat(createdTask.isRead()).isFalse();
+    assertThat(createdTask.isTransferred()).isFalse();
+
+    Task retrievedTask = taskService.getTask(createdTask.getId());
+    assertThat(retrievedTask.getCallbackInfo()).isEqualTo(callbackInfo);
+  }
+
+  @Test
+  void should_ThrowException_When_NoUserIdIsSetAndSecurityIsOn() {
+    Task task = createDefaultTask();
+
+    ThrowingCallable call = () -> taskService.createTask(task);
+    assertThatThrownBy(call).isInstanceOf(NotAuthorizedOnWorkbasketException.class);
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_ThrowException_When_CreatingTaskWithNonEmptyId() {
+    Task task = createDefaultTask();
+    ((TaskImpl) task).setId("TKI:000000000000000000000000000000000000");
+
+    ThrowingCallable call = () -> taskService.createTask(task);
+    assertThatThrownBy(call).isInstanceOf(InvalidArgumentException.class);
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_ThrowException_When_UserNotAuthorizedOnWorkbasket() throws Exception {
+    Task task = createDefaultTask();
+
+    ThrowingCallable call = () -> taskService.createTask(task);
+    NotAuthorizedOnWorkbasketException e =
+        catchThrowableOfType(call, NotAuthorizedOnWorkbasketException.class);
+    assertThat(e.getWorkbasketId()).isEqualTo(defaultWorkbasketSummary.getId());
+    assertThat(e.getCurrentUserId()).isEqualTo("user-1-1");
+    assertThat(e.getRequiredPermissions()).containsExactly(WorkbasketPermission.READ);
+    assertThatThrownBy(call).isInstanceOf(NotAuthorizedOnWorkbasketException.class);
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_NotThrowNullPointerException_When_CreatingTaskWithoutWorkbasketSummary() {
+    Task task = new TaskImpl();
+
+    ThrowingCallable call = () -> taskService.createTask(task);
+    assertThatThrownBy(call).isInstanceOf(InvalidArgumentException.class);
+  }
+
+  @WithAccessId(user = "admin")
+  @Test // Fail
+  void should_ThrowException_When_WorkbasketMarkedForDeletion() throws Exception {
+    WorkbasketSummary newWorkbasketSummary =
+        defaultTestWorkbasket().buildAndStoreAsSummary(workbasketService);
+    Task taskToPreventWorkbasketDeletion = taskService.newTask(newWorkbasketSummary.getId());
+    taskToPreventWorkbasketDeletion.setClassificationKey(defaultClassificationSummary.getKey());
+    taskToPreventWorkbasketDeletion.setPrimaryObjRef(defaultObjectReference);
+    taskToPreventWorkbasketDeletion.setOwner("user-1-2");
+    taskService.createTask(taskToPreventWorkbasketDeletion);
+    taskService.cancelTask(taskToPreventWorkbasketDeletion.getId());
+    workbasketService.deleteWorkbasket(newWorkbasketSummary.getId());
```

Comment:
Please move this line rigth bevore the call definition, because it's a part of the "When"